templates/zabbix-agent-systemd.init.erb

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Zabbix Agent
+After=syslog.target network.target
+
+[Service]
+Type=oneshot
```

Comment:
this isn't a simple script, zabbix is a daemon. so oneshot is probably wrong. forking should be correct